stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/channel/ChannelController.kt

```diff
@@ -574,7 +586,7 @@ public class ChannelController internal constructor(
             newMessage.enrichWithCid(cid)
         }
 
-        newMessage.user = domainImpl.currentUser
+        newMessage.user = domainImpl.user.value ?: return Result(ChatError("Current user null"))
```

Comment:
What if we did it as first step of such method? There is no sense to execute any logic of this method if we don' have current user